### ISSUE TITLE
Kill defender

### ISSUE THREAD

**Issue by Nobellium1** [UnamSanctam/SilentCryptoMiner#653]
Can you also add features to kill windows defender completely? Would be useful

**Comment by UnamSanctam** [comment]
I haven't actually found any real methods of doing so yet, all the ones online don't really work from my tests.

**Comment by Nobellium1** [comment]
Yeah, was testing them too, most of them are not working on 3 different pcs that I tested on.
